docs/.readthedocs.yaml

```diff
@@ -9,14 +9,11 @@ build:
   os: ubuntu-24.04
   tools:
     python: "3.13"
+  jobs:
+    install:
+      - pip install --upgrade pip
+      - pip install --group docs .
```

Comment:
I think you don't need the period `.`

<img width="904" height="297" alt="Image" src="https://github.com/user-attachments/assets/009bccbe-00b0-4ec4-93bb-5fc57bef1804" />